src/twisted/_version.py

```diff
@@ -7,5 +7,5 @@
 
 from incremental import Version
 
-__version__ = Version('Twisted', 17, 1, 0, dev=0)
+__version__ = Version('Twisted', 17, 1, 1, dev=0)
```

Comment:
The problem being solved here (and perhaps someone should put this in the ticket description explicitly) is that a development copy of Twisted trunk after 17.1.0 will not satisfy the pkg_resources version constraint `>=17.1.0`.  The point here is not that the next version will literally be `17.1.1`, but that the development version is `17.1.0+epsilon` so that it will compare as `>17.1.0`.

There are two ways to accomplish this in terms of a version specifier before we know exactly what the next release is going to be; `17.1.0.post1` or `17.1.(something greater than zero).dev`.  In discussing this with craig (which I thought was in a comment, but I guess maybe was on IRC?) we decided that `.dev` was more in the spirit of an "in development" version (since `.post` is more for "fixed release metadata, no code changes"), so `17.1.1` is where we ended up.